docs-2.0/3.ngql-guide/6.funtions-and-expressions/5.case-expressions.md

```diff
@@ -0,0 +1,161 @@
+# CASE expressions
+
+The `CASE` expression uses conditions to filter the result of an nGQL query statement. It is used in the `YIELD` or `RETURN` clause. nGQL provides two forms of `CASE` expressions just like openCypher: the simple form and the generic form.
```

Comment:
must be used with the `YIELD` or `RETURN`?

Reply:
Good one. It can be used in WHERE clauses, although not necessary. I'll use "it is usually used in the xx" here.
Thank you!